app/Resources/FOSUserBundle/views/Registration/register_content.html.twig

```diff
@@ -0,0 +1,28 @@
+{% trans_default_domain 'FOSUserBundle' %}
+
+<div class="card">
+    <div class="card-content">
+        <div class="card-title col s12">
+            <h4>{{ 'layout.register'|trans }}</h4>
+        </div>
+        <div class="row">
+            {{ form_start(form, {'method': 'post', 'action': path('fos_user_registration_register'), 'attr': {'class': 'col s12 fos_user_registration_register'}}) }}
+            <div class="row">
+                {{ form_row(form.email, { 'attr' : {'class' : 'validate'}}) }}
```

Comment:
I would appreciate a more precise feedback.
Indentation whitespace, whitespace in parameter declaration or missing whitespace somewhere?